From 8b1b3c7bb8d661b85b1d4fb72f7a3532af9535fe Mon Sep 17 00:00:00 2001
From: David Kalnischkies <david@kalnischkies.de>
Date: Sun, 7 Jan 2018 17:06:40 +0100
Subject: [PATCH] Provide hashes for uncompressed Translation-* again

Implemented in 2f34797490e328185e89972fafca4e2d15e5c2da for bug #767253
released a few months later in v3.1.17, but then broken again a few days
later in 9dd0616b78ff11123ae027016ddec0079e054cf2 by removing the
uncompressed files too early.
---
 tools/make_disc_trees.pl | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/make_disc_trees.pl b/tools/make_disc_trees.pl
index e47610f..61dadd4 100755
--- a/tools/make_disc_trees.pl
+++ b/tools/make_disc_trees.pl
@@ -652,7 +652,6 @@ sub recompress {
 	if ($filename =~ m/\/.*\/i18n\/(Translation.*)$/o &&
 		! ($filename =~ m/\/.*\/i18n\/(Translation.*gz)$/o)) {
 		system("gzip -9c < $_ >$_.gz");
-		unlink ($_);
 	}
 }	
 
-- 
2.15.1

